src/server/db/schema/forms.ts

```diff
@@ -0,0 +1,13 @@
+import {  sql } from "drizzle-orm";
+import { pgTable, text, timestamp, integer } from "drizzle-orm/pg-core"
+
+export const forms = pgTable("formSubmission", { 
```

Comment:
formSubmission might be a bit too vague should probably say it's for the feedback form, also make sure to generate a db migration with `drizzle:generate`.